scripts/skills/actives/rf_move_under_cover_skill.nut

```diff
@@ -1,5 +1,8 @@
 this.rf_move_under_cover_skill <- ::inherit("scripts/skills/skill", {
-	m = {},
+	m = {
+		__BaseActionPointCost = 0,
+		__BaseFatigueCost = 0
```

Comment:
These two member are never set in this script, unlike with what you did in kata step `onAdded`